app/controllers/lectures_controller.rb

```diff
@@ -143,11 +143,9 @@ def update
 
     @errors = @lecture.errors
 
-    # Redirect to the correct subpage
     if @lecture.valid?
       if params[:subpage].present?
-        # TODO: #860
-        redirect_to "#{edit_lecture_path(@lecture)}##{params[:subpage]}"
+        redirect_to edit_lecture_path(@lecture, tab: params[:subpage])
```

Comment:
This is something I forgot beforehand, please make sure it's working fine, i.e. when updating the lecture by submitting some new changes, you stay on the page.